src/main/frontend/index.html

```diff
@@ -545,6 +545,18 @@
       </article>
     </div>
   </div>
+<div class="tile is-parent">
+  <article class="tile is-child">
+  <label>Display Settings</label>
+      <div class="tabs is-toggle">
```

Comment:
Maybe this would be better stylistically as a [bulma select](http://bulma.io/documentation/form/select/)?